client/src/components/atoms/Stepper.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { FunctionComponent } from "react";
+
+import StepNumber from "./StepNumber"
+
+interface StepperProps {
+  steps: Array<string>;
+  selectedStep: number;
+}
+
+const Stepper: FunctionComponent<StepperProps> = (props: StepperProps) => {
+  return (
+    <div className="stepper">
+      {props.steps.map((stepName, idx) => {
+        const isSelectedStep = props.selectedStep === idx;
+        return (
+          <>
+            <div className={"step " + (isSelectedStep ? "selected-step" : "")}>
+              <StepNumber
+                isSelectedStep={isSelectedStep}
```

Comment:
Will StepNumber have an onClick event later to change selected state

Reply:
Nope, designers said no